src/agents/workflows/review_logic.py

```
@@ -36,59 +71,215 @@ def _ensure_dict_review_state(review_state) -> Dict[str, Any]:
+                if parsed:
+                    parsed_obj = SuggestionSchema(**parsed)
+        
+        # 구조화 호출에 실패했거나 지원하지 않는 경우 일반 호출 시도
+        if not parsed_obj:
+            raw_response = safe_invoke_model(base_model, prompt)
+            text = getattr(raw_response, "content", "") if hasattr(raw_response, "content") else str(raw_response)
+            parsed = _extract_json(text)
+            if parsed:
+                parsed_obj = SuggestionSchema(**parsed)
+                
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)